statchatui/src/Home/Home.jsx

```diff
@@ -6,17 +6,31 @@ const Home = ({ onStatsChange, stats }) => {
     event.preventDefault();
 
     const formData = {
-      user: event.target.user.value,
+      gamertag: event.target.gamertag.value,
       platform: event.target.platform.value,
     };
-    const user = formData.user;
+
+    try {
+      await fetch(`http://localhost:3002/users/profile`, {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(formData),
+        credentials: "include",
```

Comment:
A comment here on why we're using "include" instead of "same-origin" (the default) would be good.

Reply:
@dswcodes0 Could you address this comment please?